importers/importer.go

```diff
@@ -17,7 +17,7 @@ type Importer interface {
 	// Init will initialize each importer with a given config. This config will contain the Unmarhsalled config file specific to this plugin.
 	// It is called during initialization of an importer plugin such as setting up network connections, file buffers etc.
 	// Importers will also be responsible for returning a valid Genesis object pointer
-	Init(ctx context.Context, cfg plugins.PluginConfig, logger *logrus.Logger) (*bookkeeping.Genesis, error)
+	Init(ctx context.Context, cfg plugins.PluginConfig, logger *loggers.MT) (*bookkeeping.Genesis, error)
```

Comment:
Could you help me understand why the new interface is needed here? Does the plugin writer need to know that we've done something special with the logger?